unittests/test_versioning.py

```diff
@@ -16,39 +17,65 @@ def test_version_format(self):
         Version('1.2-dev0')
         Version('1.2-dev5')
         Version('1.2.3-dev2')
-        self.assertRaises(ValueError, Version, None)
-        self.assertRaises(ValueError, Version, '')
-        self.assertRaises(ValueError, Version, '1')
-        self.assertRaises(ValueError, Version, '1.2a')
-        self.assertRaises(ValueError, Version, 'a.b.c')
-        self.assertRaises(ValueError, Version, '1.2.3-dev')
+        with pytest.raises(ValueError):
+            Version(None)
```

Comment:
In such cases, where the statement is too short, I think that we can accept exceptionally the following:

```python
with pytest.raises(ValueError): Version(None)
```

Please do the same in other places with the `raises`.